src/docs/components/docs/configuration.jsx

```diff
@@ -81,6 +81,19 @@ export default class CmpApi extends Component {
 								by pubvendors.json.
 							</span>
 						</span>
+						<span class={style.argument}>
```

Comment:
why is the spacing so wild in these files?

Reply:
nvm it's an upstream merge i see

Reply:
project is using `tabs` I guess this is how github renders the tabs, indentation looks right on my IDE though

Reply:
also, even if it's not conforming to our standard, i'd leave this as is so it's easier to manage upstream changes in the cmp (trying to touch as little of the core of the project as possible)

Reply:
👍